test22.js

```diff
@@ -0,0 +1,5 @@
+var foo = '' ;
+var poco = '';
+if(foo  === 'bar') {
+  console.log( 'hey' )
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26941)
```suggestion
var poco = "";
```